### PR TITLE
Handle seller house duplicates and house duplicates

### PR DESCRIPTION
**Pull request by redcolorbicycle**
This change introduces logic to ensure a seller being added cannot have a duplicate house that matches any belonging to another seller.

Implemented the necessary logic to execute the adding process.

The command is added to improve the functionality of the application

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteHouseCommand.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            + "for nonexistent fields. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_HOUSING_TYPE + "HOUSING_TYPE "
+            + PREFIX_STREET + "STREET "
+            + PREFIX_BLOCK + "BLOCK "
+            + PREFIX_LEVEL + "LEVEL "
+            + PREFIX_UNITNUMBER + "UNIT NUMBER "
+            + PREFIX_POSTALCODE + "POSTAL CODE "
+            + PREFIX_PRICE + "PRICE \n"
```
Comment (zengzihui) [inline src/main/java/seedu/address/logic/commands/DeleteHouseCommand.java]: same goes to here, remove the space before "\n"

File: src/main/java/seedu/address/model/house/UniqueHouseList.java [hunk 2]
```diff
@@ -33,24 +33,20 @@ public class UniqueHouseList implements Iterable<House> {
...
-                assert index1 != -1 : "Invalid house";
-                assert index2 != -1 : "Invalid house";
-
-                String substr1 = toCheck.toString().substring(index1);
-                String substr2 = h.toString().substring(index2);
-                if (substr1.equals(substr2)) {
-                    return true;
-                }
+        for (House h: internalList) {
+            int index1 = toCheck.toString().toLowerCase().indexOf("street");
```
Comment (zengzihui) [inline src/main/java/seedu/address/model/house/UniqueHouseList.java:37]: shall change the "index1" & "index2" etc variable names to a more meaningful name.

**Review by zengzihui — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/DeleteHouseCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    public static final String MESSAGE_INVALID_HDB = "Hdb must have a block and a level!";
+    public static final String MESSAGE_INVALID_CONDOMINIUM = "Condominium must have a level!";
+    public static final String MESSAGE_INVALID_LANDED = "Landed must not have a level or block!";
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteHouseCommand
+     * and returns a DeleteHouseCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteHouseCommand parse(String args) throws ParseException {
```
Comment (KhoonSun47) [inline src/main/java/seedu/address/logic/parser/DeleteHouseCommandParser.java:45]: Adhere to SLAP if you got the time to change it

**Review by KhoonSun47 — APPROVED** [review]
LGTM, just take note of the comment!

**Pull request merged**